### PR TITLE
Add pentane molecule

### PR DESCRIPTION
**Pull request by jennyfothergill**
- [x] needs unit test

### CODE REVIEW

File: project/src/molecules/pentane_ua.py [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+import numpy as np
+
+
+class PentaneUA(mb.Compound):
+    """Create a single particle methane compound."""
+
+    def __init__(self):
+        super(PentaneUA, self).__init__()
+        # Calculate the angle between the two ports
+        angle = np.deg2rad(109.5)
```
Comment (ramanishsingh) [inline project/src/molecules/pentane_ua.py:13]: Should the angle be =CHx-(CH2)-CHy angle i.e. 114 degrees?

Comment (jennyfothergill) [reply project/src/molecules/pentane_ua.py]: Sure!

File: project/src/molecules/pentane_ua.py [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+        ch2.add(mb.Particle(name="_CH2"))
+        ch2.add(mb.Port(anchor=ch2[0]), "up")
+        ch2["up"].translate([x, y, z])
+        ch2.add(mb.Port(anchor=ch2[0], orientation=vec), "down")
+        ch2["down"].translate(vec)
+
+        pentane = mb.recipes.Polymer(
+            monomers=[ch2], end_groups=[ch3, mb.clone(ch3)]
+        )
+        pentane.build(n=5)
```
Comment (ramanishsingh) [inline project/src/molecules/pentane_ua.py:40]: Total 5 beads for pentane. So 3 monomer beads. I think n should be 3.

Comment (jennyfothergill) [reply project/src/molecules/pentane_ua.py:40]: oh my gosh 🤦‍♀️ how did I miss that... Thanks, @ramanishsingh !

File: project/tests/test_pentane_ua.py [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+    """Tests to ensure PentaneUA behaves as expected."""
+
+    def test_creation(self):
+        pent = PentaneUA()
+        assert isinstance(pent, mb.Compound)
+        assert pent.n_particles == 5
+        assert pent.n_bonds == 4
+        assert pent.name == "PentaneUA"
+        assert all([p.name in ["_CH2", "_CH3"] for p in pent.particles()])
+        assert len(pent.labels) == 1
```
Comment (ramanishsingh) [inline project/tests/test_pentane_ua.py:19]: Check one bond length and one angle as well? Might not be required though.

```python 
np.linalg.norm(pentane.xyz[3]-pentane.xyz[0])*10
```
```python


a = pentane.xyz[0]
b = pentane.xyz[1]
c = pentane.xyz[2]

ba = a - b
bc = c - b

cosine_angle = np.dot(ba, bc) / (np.linalg.norm(ba) * np.linalg.norm(bc))
angle = np.arccos(cosine_angle)

print(np.degrees(angle))
```

Comment (ramanishsingh) [reply project/tests/test_pentane_ua.py:19]: Angle calculation code taken from here: https://stackoverflow.com/questions/35176451/python-code-to-calculate-angle-between-three-point-using-their-3d-coordinates

**Comment by justinGilmer** [comment]
Just updated with some of ramanish's suggestions as well as some parametrization checks. Once these pass, LGTM!

**Review by justinGilmer — APPROVED** [review]

**Review by ramanishsingh — APPROVED** [review]

**Pull request merged**